docs/conf.py

```diff
@@ -15,20 +15,20 @@
 import os
 import sys
 
-sys.path.append(
-    os.path.join(os.path.dirname(__file__), '..', '..', 'src')
-)
+# sys.path.append(os.path.join(os.path.dirname(__file__), "..", "src"))
+#
+# import fixate
 
 # -- Project information -----------------------------------------------------
 
-project = 'Fixate'
-copyright = '2018, Clint Lawrence, Ryan Parry-Jones'
-author = 'Clint Lawrence'
+project = "Fixate"
+copyright = "2024, Clint Lawrence, Ryan Parry-Jones"
+author = "Clint Lawrence"
 
 # The short X.Y version
-version = ''
+version = "" #fixate.__version__
```

Comment:
Do we need to specifiy `version` and `release` at all if the default is `''`?

Reply:
Also, I'm bumbling through the sphink documentation so try and understand all this better, so probably a lot of detail/understanding that I'm missing at this stage.

Reply:
I've changed this back to it uses `fixate.__version__` to set the `version` and `release` attributes.